src/test/java/decodex/commands/SelectCommandTest.java

```diff
@@ -0,0 +1,62 @@
+package decodex.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import decodex.data.DataManager;
+import decodex.data.exception.CommandException;
+import decodex.data.exception.DataManagerException;
+import decodex.data.exception.UnknownModuleException;
+import decodex.modules.ModuleManager;
+import decodex.ui.Ui;
+import org.junit.jupiter.api.Test;
+
+public class SelectCommandTest {
+
+    @Test
+    public void run_validModuleName_success() throws UnknownModuleException, CommandException, DataManagerException {
+        DataManager dataManager = new DataManager();
+        ModuleManager moduleManager = new ModuleManager();
+        Ui ui = new Ui();
+
+        DataCommand dataCommand = new DataCommand("hello world");
+        dataCommand.run(dataManager, moduleManager, ui);

```

Comment:
Instead of relying on another command, perhaps its better to insert the data directly into the `dataManager` instead?